sdk/core/core-arm/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,212 @@
+Third Party Notices for core-arm
```

Comment:
core-arm doesn't seem to be producing a browser bundle. Its only dependency, core-http, is external, and therefore this file can be deleted I believe.